### DISCUSSION TITLE
티스토리 OPEN API 종료에 따른 동글 티스토리 지원 기능 삭제

### DISCUSSION THREAD

**Discussion by echo724** [woowacourse-teams/2023-dong-gle#613] [기능 제안]
## 근거
[https://notice.tistory.com/2664](https://notice.tistory.com/2664)

## 과정
- '파일 첨부 > 글 관련 기능 > 댓글 관련 기능 > 그 외 기능' 순으로 종료되며, 2024년 2월 내에 전부 종료가 된다고 합니다.
- 사실 파일 첨부 기능이 종료되면 이미지 업로드가 불가능하기 때문에 핵심 기능이 안된다고 생각이 됩니다.
- 따라서 현재 기준으로 동글의 티스토리 블로그 지원 기능을 제거해나가고자 합니다.
- 다만 다른 플랫폼을 연결할때 코드를 재활용이 가능할 수 도 있기 때문에 API만 막아두는 형식으로 하는건 어떨까 싶습니다.

### 백엔드
**티스토리와 관련된 API 엔드포인트 비활성화**
1. 컨트롤러에 엔드포인트 메서드 제거
2. 마이페이지에서 티스토리 연결 API 제거

### 프론트엔드
**1. 티스토리 OPEN API 종료에 따른 공지? Notification 활성화**
아래 두가지 방식이 생각나긴 하는데 이 중에 괜찮은 방법이나 다른 방법이 있다면 알려주세요.

1. Notification Badge
<img width="219" alt="Screenshot 2024-01-04 at 16 38 12" src="https://github.com/woowacourse-teams/2023-dong-gle/assets/50057022/56a54649-2442-4892-b935-4c9857fd3f45">

- 위의 사진의 메뉴바 아래에 공지사항이란 항목을 만들고 거기에 Badge 형태의 알림 및 공지사항 깃헙 위키 페이지로 연결
![image](https://github.com/woowacourse-teams/2023-dong-gle/assets/50057022/6d4cdaab-23d2-49b3-8763-d4e442e5b71b)
- 요런 뱃지를 ... 아이콘 옆에 두어 사용자가 동글 화면에 들어오면 뭔가 알림이 있다는걸 알고 공지사항으로 연결될 수 있도록

2. 배너 알림
- UI 상으로 Footer나 Header 영역에 티스토리 기능 비활성화 안내

**2. 티스토리 발행과 관련된 버튼 제거**
- 아래 두 부분 제거
<img width="289" alt="Screenshot 2024-01-04 at 16 44 28" src="https://github.com/woowacourse-teams/2023-dong-gle/assets/50057022/7a39a156-129c-42b0-b703-ca680538d527">
<img width="936" alt="Screenshot 2024-01-04 at 16 44 34" src="https://github.com/woowacourse-teams/2023-dong-gle/assets/50057022/9ec64786-4f8d-47b5-a6a8-2fac762f2769">

## 작업 시작일
- 1월 2째주부터 해서 천천히 작업을 해나가는게 어떨까 싶습니다.

**Comment by yogjin** [comment]
아 이게뭔가요...
미디움 API도 12월 28일 부로 무기한 작업에 들어갔다고 하네요..
https://medium.statuspage.io/
![image](https://github.com/woowacourse-teams/2023-dong-gle/assets/33623078/9991a910-f649-409e-8b7c-57570a0c7bbf)

**Comment by echo724** [comment]
ㅠㅡㅠ 아무래도 서비스 종료에 가까이 온 것 같습니다.
이를 공지할 수 있는 방법이 필요할 것 같네요..
